lib/logthrdestdrv.c

```diff
@@ -32,8 +32,8 @@ log_threaded_dest_driver_format_seqnum_for_persist(LogThrDestDriver *self)
 {
   static gchar persist_name[256];
```

Comment:
if it's only an sprintf, then I would use g_strdup_printf(), but I guess
the problem is allocation and how to free this allocation.

-- 
Bazsi

On Wed, Jul 13, 2016 at 1:10 PM, Budai Laszlo <notifications@github.com>
wrote:

> In lib/logthrdestdrv.c
> <https://github.com/balabit/syslog-ng/pull/665#discussion_r70605993>:
>
> > @@ -32,8 +32,8 @@ log_threaded_dest_driver_format_seqnum_for_persist(LogThrDestDriver *self)
> >  {
> >    static gchar persist_name[256];
>
> note : maybe it is time to using a GString instead of the static buffer?
> (or, at least we should tell the user that the persist name length has a
> max length...).
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/balabit/syslog-ng/pull/665/files/02536ffae49a761b9e76709963203a2eb11057b0..3571c7fda90bface84257cd23df7a87c47b455c9#r70605993>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AArlduEEbuzQmB-qe_ib2htXR1YDzaaxks5qVMecgaJpZM4F2VRA>
> .
>